### DISCUSSION TITLE
Apply effects to all layers

### DISCUSSION THREAD

**Discussion by octvs** [WillPower3309/swayfx#347] [Q&A]
Hey,

I have repeated lines on my config for applying same effects
```swayconfig
layer_effects "waybar" blur enable; shadows enable; corner_radius 10
layer_effects "wob" blur enable; shadows enable; corner_radius 10
```
Is there a way to define this to all layers, a la `layer_effects *`?

Thanks for the awesome work!

**Comment by markstos** [comment]
Is this way my 1Password app window has a drop shadow, but if 1Password pops up a small window asking for my password, it does not have a drop shadow? Because it's a different type of layer? I tried this method to find the layer namespace of that pop-up while it was open:

```
swaymsg -r -t get_outputs | jq '.[0].layer_shell_surfaces | .[] | .namespace'
```

But the only namespace returned was `wallpaper`. 

Maybe it's a special type of a layer because it's collecting a password.
